i18n/routing.ts

```diff
@@ -0,0 +1,15 @@
+import { defineRouting } from 'next-intl/routing';
```

Comment:
This is required by `next-intl` to establish which locales are supported for routing